CentralManagement/ChocoCCM Command Reference/New-CCMOutdatedSoftwareReport.md

```diff
@@ -0,0 +1,41 @@
+---
+external help file: ChocoCCM-help.xml
+Module Name: ChocoCCM
+online version:
+schema: 2.0.0
+---
+
+# New-CCMOutdatedSoftwareReport
+
+## SYNOPSIS
+Create a new Outdated Software Report in Central Management

```

Comment:
Hrm. I'm wondering whether we're better off just doing `New-CcmReport -Type OutdatedSoftware` or something. This cmdlet name is way too long.